.github/workflows/publish.yml

```diff
@@ -23,7 +23,7 @@ jobs:
       - name: Setup Node Environment ⬢
         uses: actions/setup-node@v1
         with:
-          node-version: 16
+          node-version: 18
```

Comment:
should we be using a version "matrix" now? it might be nice to support testing on 16-18?

Reply:
Thats a good idea.  If you're good with it lets scope a different piece of work build better tests and run them on all the different combos of system + node version

Reply:
Awesome, we should probably just add that matrix setup to ts-template and use that across the board for tests etc.